.claude/agent-memory/archgate-developer/project_pr_review_thread_triage.md

```
@@ -1,13 +1,11 @@
 ---
 name: project-pr-review-thread-triage
-description: How to distinguish already-resolved vs genuinely outstanding PR review comments — REST API doesn't expose resolution state
+description: Find genuinely outstanding PR review comments — the REST API does not expose thread resolution state, only GraphQL does
 metadata:
   type: project
 ---
 
-**`gh api repos/<owner>/<repo>/pulls/<n>/comments` (REST) does NOT expose whether a review comment thread is resolved.** A stale CodeRabbit/reviewer comment from an earlier commit stays in that endpoint's output forever, indistinguishable from a live, unaddressed one — reading it naively re-litigates already-fixed findings.
-
-**Fix: use the GraphQL `reviewThreads` field**, which has `isResolved` and `isOutdated`:
+The REST API does not report whether a review thread is resolved, so `gh pr view --json comments` lists finished threads as if they were open. Use the GraphQL `reviewThreads` field and filter on `isResolved`:
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)